Tests for esmvaltool/diag_scripts/mlr failing

The nightly tests are failing for esmvaltool/diag_scripts/mlr

Example stack trace from 
`tests.integration.diag_scripts.mlr.test_custom_sklearn_functions::test_fit_and_score_weighted_no_weights[scorer0-2.5]`


```
>               raise ValueError(
                    f"Invalid parameter {key!r} for estimator {self}. "
                    f"Valid parameters are: {local_valid_params!r}."
                )
E               ValueError: Invalid parameter 'normalize' for estimator LinearRegression(). Valid parameters are: ['copy_X', 'fit_intercept', 'n_jobs', 'positive'].

/opt/conda/envs/esmvaltool/lib/python3.10/site-packages/sklearn/base.py:205: ValueError
```

CircleCI run: https://app.circleci.com/pipelines/github/ESMValGroup/ESMValTool/8430/workflows/690c929b-3861-4104-a742-d8360551a8b2/jobs/62199

--- a/tests/integration/diag_scripts/mlr/test_custom_sklearn_functions.py
+++ b/tests/integration/diag_scripts/mlr/test_custom_sklearn_functions.py
@@ -640,7 +640,7 @@ def test_fit_and_score_weighted_no_weights(scorer, output):
     clf = LinearRegression()
     fit_and_score_weighted_args = [clf, X_DATA, Y_DATA, scorer, TRAIN, TEST]
     fit_and_score_weighted_kwargs = {
-        'parameters': {'normalize': False},
+        'parameters': {'copy_X': True},
         'fit_params': None,
     }
 
@@ -666,7 +666,7 @@ def test_fit_and_score_weighted_weights(scorer, output):
     clf = LinearRegression()
     fit_and_score_weighted_args = [clf, X_DATA, Y_DATA, scorer, TRAIN, TEST]
     fit_and_score_weighted_kwargs = {
-        'parameters': {'normalize': False},
+        'parameters': {'copy_X': True},
         'fit_params': {'sample_weight': SAMPLE_WEIGHTS},
         'sample_weights': SAMPLE_WEIGHTS,
     }

